anaconda.py

```diff
@@ -119,6 +119,25 @@ def parse_arguments(argv=None, boot_cmdline=None):
     return (namespace, ap.removed_no_inst_bootargs)
 
 
+def print_dracut_errors(stdout_logger):
+    """Print important errors from Dracut to user before starting Anaconda.
+
+    :param stdout_logger: python logger to stdout
+    """
+    from pyanaconda.core.constants import DRACUT_ERRORS_PATH
+
+    try:
+        with open(DRACUT_ERRORS_PATH, "rt") as fd:
+            section_name = "Installer errors encountered during boot"
+            msg = "\n{:#^70}\n{}\n{:#^70}".format(  # add starting \n because timestamp
+                " " + section_name + " ",           # start of the section
+                "".join(fd.readlines()),            # errors from Dracut
+                " " + section_name + " end ")       # end of the section
+            stdout_logger.warning(msg)
```

Comment:
On a second thought, I am actually not sure if `stdout_logger` is the right kind of output?

Reply:
I was also thinking about that, reason why I decided to use it is that we are already using it for printing errors on this place. I wanted to be consistent.